Scoop/Controller/Ride Posting/PostRideLocationViewController.swift

```diff
@@ -22,8 +22,7 @@ class PostRideLocationViewController: UIViewController {
     private let fieldSpace = 40
     private let labelSpace = 5
     private let textFieldSpace = 20
-    
-    private var ride = Ride()
+    private var ride = try! Ride()
```

Comment:
Do we have to force unwrap here? This looks dangerous

Reply:
Should have documented this, but I had force unwrapped it because the Ride model was giving me issues but will fix this in my next PR

Reply:
Can you add that documentation in a "// TODO:"